BootloaderCorePkg/Tools/GenerateKeys.py

```diff
@@ -120,14 +122,18 @@ def generate_rsa_pub_keys (openssl_path, key_dir, key_size, replace_key):
                     continue
 
             print("Generating OS Private Key %s" % priv_key_name)
-            cmd = '%s genrsa -F4 -out %s %s' % (openssl_path, priv_key_name, key_size)
-            run_process (cmd.split())
+            cmd = 'genrsa -F4 -out %s %s' % (priv_key_name, key_size)
+            cmd = cmd.split()
+            cmd.insert(0, '%s' % (openssl_path))
+            run_process (cmd)
 
             # Extract public key from private key
             pub_key_name = '%s/%s_Pub_RSA%s.pem' % (key_dir, key_file_name, key_size)
             print("Generating OS Public Key %s" % pub_key_name)
-            cmd = '%s rsa -pubout -in %s -out %s' % (openssl_path, priv_key_name, pub_key_name)
-            run_process (cmd.split())
+            cmd = 'rsa -pubout -in %s -out %s' % (priv_key_name, pub_key_name)
+            cmd = cmd.split()
+            cmd.insert(0, '%s' % (openssl_path))

```

Comment:
If there is a more optimal way to split the text up without modifying the openssl part please let me know, this is just what I came up with.

Reply:
I guess there is space in openssl_path, so that run_process doesn't really get the file extension, right?
If openssl_path is got from get_openssl_path(), we only need change get_openssl_path().

Reply:
I don't think that will help in this case, in Azure environment it looks like openssl is located in "C:\Program Files", how can we remove the space there? Maybe I am not understanding though, let me cancel this PR so that you can get it done more quickly than me, I don't want to hold this up.

Reply:
I think the failure should be caused by run_process adding ".exe" wrongly. we could enhance run_process to avoid that.